LL/dal_corpri_arad.ttl

```diff
@@ -1168,336 +1601,1145 @@
+    owl:sameAs <#Cobthach>.
 
-<#Fothaid-db7f68ea>
+<#Damain-e41ef7f0>
     a foaf:Person;
-    irishRel:genName "Fothaid";
-    irishRel:nomName "Fothad";
-    owl:sameAs <#Fhothaid-7fb47155>.
+    irishRel:nomName "Daman";
+    irishRel:genName "Damain";
+    rdfs:comment ".h. Damain".
 
-# Do shíl Conaill dano teora ingena Eogain mc Conaill con-gabsat
-#  (http://www.dil.ie/12134) Cill n-Garraisc. (for the seed of conall
-#  moreover three daughters of Eogan son of conall who contained Cill
-#  n-Garraisc) - CY
+<#Thian>
+    a foaf:Person;
+    irishRel:nomName "Tian";
+    irishRel:genName "Thian";
+    rdfs:comment ".h. Thian".
 
-<#Findchad-d6245ea7>
-# same as <#findchad> - CY?
+<#Conall-4473eb10>
     a foaf:Person;
-    irishRel:nomName "Findchad".
+    irishRel:nomName "Conall";
+    rdfs:comment "a quo .h. Con Anróit";
+    owl:sameAs <#Conall-55dab9c0>.
 
-<#Nia>
+<#ConAnróit>
     a foaf:Person;
-    irishRel:nomName "Nia";
-    rel:childOf <#BecceBernsa>.
+    irishRel:genName "Con Anróit";
+    irishRel:nomName "Cú Anróit";
+    rdfs:comment ".h. Con Anróit".
 
-<#BecceBernsa>
+<#Glasderc-beaf6c10>
     a foaf:Person;
-    irishRel:nomName "Becce Bernsa";
-    rel:childOf <#Eogain-35ac20f0>.
+    irishRel:nomName "Glasderc";
+    rdfs:comment "a quo .h. Glasdeirc";
+    owl:sameAs <#Glasderc>.
 
-<#Eogain-35ac20f0>
+<#Fubaide>
     a foaf:Person;
-    irishRel:nomName "Eogain";
-    rel:childOf <#Éitchen>.
+    irishRel:nomName "Fubaide";
+    owl:sameAs <#Furbbaide>;
+    rdfs:comment "a quo .h. Fuachon .h. Baetain .&c".
 
-<#Éitchen>
+<#Fuachon>
     a foaf:Person;
-    irishRel:nomName "Éitchen";
-    rel:childOf <#Baeth>.
+    irishRel:genName "Fuachon";
+    rdfs:comment ".h. Fuachon".
 
-<#Baeth>
+<#Baetain>
     a foaf:Person;
-    irishRel:nomName "Baeth";
-    rel:childOf <#Maithgein>.
+    irishRel:genName "Baetain";
+    irishRel:nomName "Baetan";
+    rdfs:comment ".h. Baetain".
 
-<#Maithgein>
+<#FergusLamderg>
     a foaf:Person;
-    irishRel:genName "Maithgein";
-    irishRel:nomName "Maithgen";
-    rel:childOf <#Findchada>.
+    irishrel:nomName "Fergus Lamderg";
+    rel:childof <#Fothaid-49b630b0>;
+    owl:sameAs <#FergusLaebderg>;
+    rdfs:comment "A quo etiam .h. Chaili .h. Flathechain .h. Fidrechain .h. Chathain .h. Gaimdechair .h. Dobrain. Síl Cruachi for Gabrán.".
 
-<#Findchada>
+<#Fothaid-49b630b0>
+    owl:sameAs <#Fhothaid-7fb47155>.
+
+<#Coemgen>
     a foaf:Person;
-    irishRel:genName "Findchada";
-    irishRel:nomName "Findchad";
-    owl:sameAs <#Findchad-d6245ea7>.
+    irishRel:nomName "Coemgen";
+    rdfs:comment "Glinni da Locha";
+    rdfs:seeAlso <https://www.wikidata.org/wiki/Q737198>;
+    rel:descendantOf <#FergusLamderg>.
 
-<#Garrchon>
+<#Eogain-f478a320>
     a foaf:Person;
-    irishRel:numChild 7;
-    irishRel:nomName "Garrchon".
-# sameAs <#Garrchu>? - CY
+    irishrel:genName "Eogain";
+    rdfs:comment "Aird Sratha";
+    rdfs:seeAlso <https://www.wikidata.org/wiki/Q8078337>;
```

Comment:
I did a search for "https://www.wikidata.org/wiki" across the project and there were actually quite a few instances. So I replaced them all with "https://www.wikidata.org/entity". Hence the large number of files that will now be involved in this pull request...